src/cloud_what/providers/aws.py

```diff
@@ -133,6 +139,39 @@ def is_likely_running_on_cloud(self) -> float:
 
         return probability
 
+    @staticmethod
+    def fix_rhui_url_template(repo: Any, region: str) -> None:
+        """
+        Try to fix URL of RHUI repository on AWS
+        :param repo: DNF object of repository
+        :param region: string representing region
+        :return: None
+        """
+        if region in REDIRECT_MAP:
+            region = REDIRECT_MAP[region]
+
+        if repo.baseurl:
+            repo.baseurl = tuple(
+                url.replace('REGION', region, 1) for url in repo.baseurl
+            )
+        elif repo.mirrorlist:
+            repo.mirrorlist = repo.mirrorlist.replace('REGION', region, 1)
+        else:
+            raise ValueError(f"RHUI repository {repo.name} does not have any url")
+
+    @staticmethod
+    def rhui_repos(base: Any) -> Any:
+        """
+        Generator of RHUI repositories. Could be used e.g. in for loop
+        :param base: DNF base
+        :return: Yields RHUI repositories
+        """
+        for repo_name, repo in base.repos.items():
+            # TODO: we need more reliable mechanism for detection of RHUI repository, because
```

Comment:
Is this captured somewhere besides this comment?

Reply:
We can create card for this task.